src/utils/getCliFeedJson.ts

```diff
@@ -23,10 +25,20 @@ export type cliFeedJsonResponse = {
     }
 };
 
-export async function getCliFeedJson(): Promise<cliFeedJsonResponse> {
-    const funcJsonOptions: request.OptionsWithUri = {
-        method: 'GET',
-        uri: funcCliFeedUrl
-    };
-    return <cliFeedJsonResponse>JSON.parse(await <Thenable<string>>request(funcJsonOptions).promise());
+export async function tryGetCliFeedJson(): Promise<cliFeedJsonResponse | undefined> {
+    try {
+        // tslint:disable-next-line:no-unsafe-any
+        return await callWithTelemetryAndErrorHandling('azureFunctions.tryGetCliFeedJson', ext.reporter, undefined, async function (this: IActionContext): Promise<cliFeedJsonResponse> {
+            this.properties.isActivationEvent = 'true';
+            this.suppressErrorDisplay = true;
+            const funcJsonOptions: request.OptionsWithUri = {
+                method: 'GET',
+                uri: funcCliFeedUrl
+            };
+            return <cliFeedJsonResponse>JSON.parse(await <Thenable<string>>request(funcJsonOptions).promise());
+        });
+
+    } catch (error) {
```

Comment:
nit: This try/catch is redundant since callWithTelemetryAndErrorHandling swallows errors and returns undefined by default